src/main/java/nl/thehyve/ocdu/OcduApplication.java

```diff
@@ -8,12 +8,18 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+import org.springframework.scheduling.annotation.EnableScheduling;
 
 import java.util.Date;
 
 @SpringBootApplication
+@EnableAutoConfiguration
```

Comment:
I am not sure if this is necessary - AFAIK @SpringBootApplication annotation already contains it